src/components/dex/Pair.vue

```diff
@@ -108,6 +107,20 @@ export default {
       return value;
     },
 
+    setMarketData() {
+      this.marketData = _.reduce(this.tickerData, (marketData, market, ticker) => {
+        const close24Hour = this.getClose24Hour(ticker);
+        const change24Hour = this.getChange24Hour(close24Hour, market.open24hr);
```

Comment:
is it possible for market.open24hr to be undefined? If so I think it will lead to an exception in `this.getChange24Hour` called here.